modules/core/shared/src/main/scala/lucuma/core/enums/CatalogName.scala

```diff
@@ -5,15 +5,14 @@ package lucuma.core.enums
 
 import lucuma.core.util.Enumerated
 
-sealed trait CatalogName extends Product with Serializable
+sealed abstract class CatalogName(val tag: String) extends Product with Serializable
 
 object CatalogName {
-  case object Simbad extends CatalogName
-  case object Horizon extends CatalogName
-  case object Gaia extends CatalogName
+  case object Simbad extends CatalogName("simbad")
+  case object Gaia extends CatalogName("gaia")
```

Comment:
`Horizon` should have at least been `Horizons` but did we use it for anything?

Reply:
These are used for `CatalogInfo` which is for sidereal targets, which don't exist in Horizons, so I think it's unused.